src/ProposalPayload.sol

```diff
@@ -1,17 +1,39 @@
-// SPDX-License-Identifier: AGPL-3.0-only
+// SPDX-License-Identifier: MIT
+
+/*
+   _      ΞΞΞΞ      _
+  /_;-.__ / _\  _.-;_\
+     `-._`'`_/'`.-'
+         `\   /`
+          |  /
+         /-.(
+         \_._\
+          \ \`;
+           > |/
+          / //
+          |//
+          \(\
+           ``
+     defijesus.eth
+*/
+
 pragma solidity ^0.8.15;
 
-import {AaveMisc} from "@aave-address-book/AaveMisc.sol";
 import {AaveV2Ethereum} from "@aave-address-book/AaveV2Ethereum.sol";
 
 /**
- * @title <TITLE>
+ * @title wETH Interest Rate Curve - Ethereum v2
  * @author Llama
- * @notice <DESCRIPTION>
- * Governance Forum Post:
- * Snapshot:
+ * @notice Upgrade wETH interest rate on Ethereum v2 to match Ethereum v3 Liquidity Pool.
+ * Governance Forum Post: https://governance.aave.com/t/arfc-weth-wsteth-interest-rate-curve-ethereum-network/11372
+ * Snapshot: https://snapshot.org/#/aave.eth/proposal/0x9ae28e9c82c5fc0d24cf1df788094e959d99f906d11b89e455a60ee16b071d6f
  */
 contract ProposalPayload {
+    address public constant WETH = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
```

Comment:
WETH should come from address-book.

Reply:
cant find WETH on the address book, maybe I'm missing something